libnetwork/networkdb/networkdbdiagnostic.go

```diff
@@ -271,22 +249,17 @@ func dbDeleteEntry(ctx interface{}, w http.ResponseWriter, r *http.Request) {
 	nid := r.Form["nid"][0]
 	key := r.Form["key"][0]
 
-	nDB, ok := ctx.(*NetworkDB)
-	if ok {
-		err := nDB.DeleteEntry(tname, nid, key)
-		if err != nil {
-			logger.WithError(err).Error("delete entry failed")
-			diagnostic.HTTPReply(w, diagnostic.FailCommand(err), json)
-			return
-		}
-		logger.Info("delete entry done")
-		diagnostic.HTTPReply(w, diagnostic.CommandSucceed(nil), json)
+	err := nDB.DeleteEntry(tname, nid, key)
+	if err != nil {
```

Comment:
Micro-nit; this could be inlined;

```go
if err := nDB.DeleteEntry(tname, nid, key); err != nil {
```

Reply:
This comment gets hidden when I hide whitespace in the diff view, so I'm inclined to leave the change whitespace-only to keep the noise down for reviewers.

Reply:
Yeah, for sure not a blocker in either way.